src/arch/arm/32/kernel/vspace.c

```diff
@@ -497,7 +497,7 @@ BOOT_CODE cap_t create_it_address_space(cap_t root_cnode_cap, v_region_t it_v_re
 
     /* create PD cap */
     copyGlobalMappings(PDE_PTR(rootserver.vspace));
-    cleanCacheRange_PoU(rootserver.vspace, rootserver.vspace + (1 << seL4_PageDirBits) - 1,
+    cleanCacheRange_PoU(rootserver.vspace, rootserver.vspace + BIT(seL4_PageDirBits) - 1,
```

Comment:
```suggestion
    cleanCacheRange_PoU(rootserver.vspace, rootserver.vspace + MASK(seL4_PageDirBits),
```
here is also a MASK macro that does this.